guides/v2.2/rest/bulk-endpoints.md

```diff
@@ -11,7 +11,7 @@ functional_areas:
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
 {:.bs-callout .bs-callout-tip}
-Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing.
+Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing, after configure the message queues. For more details see: [Manage message queues]({{ page.baseurl }}config-guide/mq/manage-message-queues.html).
```

Comment:
I don't think that the manage message queues is the best place to point.

Elsewhere in the REST API Guide:
https://devdocs.magento.com/guides/v2.3/rest/asynchronous-web-endpoints.html
```Use the bin/magento queue:consumers:start async.operations.all command to start the consumer that handles asynchronous and bulk API messages.```

Perhaps it's enough to include this note in the bulk-endpoints topic with an additional link to the [Start message queue consumers]https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-queue.html) reference topic?

The Extensions Guide provides the details for implementing the [Bulk Operations](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/message-queues/implement-bulk.html).

There is also a tutorial about bulk endpoints: https://devdocs.magento.com/guides/v2.3/rest/tutorials/bulk-configurable-product/config-product-intro.html

It seems like they can try the command, if it fails, then they could list the consumers.  If the `async.operations.all` consumer isn't available, then you would have to figure out how to enable it. See [Bulk operations] (https://devdocs.magento.com/guides/v2.3/extension-dev-guide/message-queues/bulk-operations.html) in the Extension Guide.